docs/samediff/templates/model-import.md

```diff
@@ -1 +1,61 @@
-# Getting started: importing TensorFlow and ONNX models into SameDiff
\ No newline at end of file
+---
+title: Getting started: importing TensorFlow models into SameDiff
+short_title: Model import
+description: importing TensorFlow models into SameDiff
+category: SameDiff
+weight: 3
+---
+
+# Getting started: importing TensorFlow models into SameDiff
+
+## What models can be imported into SameDiff
+
+Currently SameDiff supports the import of TensorFlow frozen graphs through the various SameDiff.importFrozenTF methods. 
+TensorFlow documentation on frozen models can be found [here](https://www.TensorFlow.org/guide/saved_model#the_savedmodel_format_on_disk). 
+
+    import org.nd4j.autodiff.SameDiff.SameDiff;
+    
+    SameDiff sd = SameDiff.importFrozenTF(modelFile);
+    
+ ## Finding the model input/outputs and running inference
+ 
+ After you import the TensorFlow model there are 2 ways to find the inputs and outputs. The first method is to look at the output of
+ 
+     sd.summary();
+     
+ Where the input variables are the output of no ops, and the output variables are the input of no ops.  Another way to find the inputs is
+ 
+      List<String> inputs = sd.inputs();
+    
+ To run inference use:
+ 
+    INDArray out = sd.batchOutput()
+        .input(inputs, inputArray)
+        .output(outputs)
+        .execSingle();
```

Comment:
"For multiple outputs, use exec() instead of execSingle(), to return a `Map<String,INDArray>` of outputs instead.
Alternatively, you can use methods such as `SameDiff.output(Map<String, INDArray> placeholders, String... outputs)` to get the same output."